src/GraphResponseHandler.ts

```diff
@@ -101,6 +101,7 @@ export class GraphResponseHandler {
 			return Promise.resolve();
 		}
 		let responseValue: any;
+		const contentType = rawResponse.headers.get("Content-type");
```

Comment:
I've seen a few headers at a couple of places, constants?

Reply:
Can you please elaborate the question?

Reply:
https://github.com/microsoftgraph/msgraph-sdk-javascript/search?l=TypeScript&q=%22Content-type%22
This constant (more generally all the headers keys) is defined multiple times instead of once as a constant. Would it be worth refactoring that?

Reply:
I will have to look into this and check if that is possible. Will scope this outside the linting task.

Reply:
out of curiosity, how do we capture the things we say we'll do later today? issue in github? somewhere else?